### ISSUE TITLE
qgis-dev crashes upon start with no apparent reason

### ISSUE THREAD

**Issue by EranGeo** [qgis/QGIS#49546]
### What is the bug or the crash?

Opening the qgis-dev installation results in a crash. This is happaninng for the last 3 weeks or so. I tried to reinstall all dev componants using osgeo-setup file but it didn't help.

### Steps to reproduce the issue

1. Using osgeo-setup.exe, download and install the latest qgis-dev version.
2. Open qgis-dev

## Report Details

**Crash ID**: [Not available](https://github.com/qgis/QGIS/search?q=Not available&type=Issues)

**Stack Trace**
No stack trace is available.

**QGIS Info**
QGIS Version: 3.27.0-Master
QGIS code revision: fc5faa84bf
Compiled against Qt: 5.15.3
Running against Qt: 5.15.3
Compiled against GDAL: 3.6.0dev-023d25cf45
Running against GDAL: 3.6.0dev-023d25cf45

**System Info**
CPU Type: x86_64
Kernel Type: winnt
Kernel Version: 10.0.17763

### Supported QGIS version

- [ ] I'm running a supported QGIS version according to the roadmap.

### New profile

- [ ] I tried with a new QGIS profile

### Additional context

In one occasion when I was updating my QGIS installtion stack (ltr, latest and dev), I think the dev version was running and needed to be closed in order for the installer to finish.

**Comment by agiudiceandrea** [comment]
I think this issue is related to the OSGeo4W packaging system and not to QGIS itself, so it would be better to report this issue at https://trac.osgeo.org/osgeo4w/.

**Comment by nicogodet** [comment]
@EranGeo Can you check this similar issue https://github.com/qgis/QGIS/issues/49507 ?

**Comment by EranGeo** [comment]
@nicogodet , yes! I also have the same qgis [installation stack](https://github.com/qgis/QGIS/issues/49507#issuecomment-1203504690) like in ticket #49507 . The stack was installed using osgeo4w installer.

 The only difference is that I'm using MS Windows shortcut link located at the taskbar. This shortcut has a `Start in` variable, that in my case is `C:\Users\<os_user_name>\Documents`
![qgis-dev shortcut properties](https://user-images.githubusercontent.com/47955227/182586910-604e4b07-b110-42be-af08-f7eb490a13cd.png)

**Comment by agiudiceandrea** [comment]
@EranGeo just to clarify: does the crash happen for you during the starting process of qgis-dev when "Setting up the GUI" is displayed under the splash screen? Does the crash occur only if there is a file or folder with non-ASCII characters in the directory specified in the "Start in" property of the QGIS shortcut link?

**Comment by EranGeo** [comment]
> @EranGeo just to clarify: does the crash happen for you during the starting process of qgis-dev when "Setting up the GUI" is displayed under the splash screen?

Yes.

![splash and crash screens](https://user-images.githubusercontent.com/47955227/182591563-8e1153f2-2f34-4461-84c9-717ba7543341.png)


>Does the crash occur only if there is a file or folder with non-ASCII characters in the directory specified in the "Start in" property of the QGIS shortcut link?

The path itself **does not** contain non-ASCII characters. But my OS username **does** contain dot between first and last name, e.g. `C:\Users\john.smith\Documents`.

I did had some non-ASCII characters in some files names - files that were present in the shortcut's `Start in` directory. Removing them, solved the issue! I did additional test and changed one of the folders present inside that directory, to be named with non-ascii characters. Doing so, the crash was once again reproduced!
